aisim/atoms.py

```diff
@@ -70,22 +73,33 @@ def time(self, value):
         self._time = value
 
     @property
-    def state_vectors(self):
+    def state_kets(self):
         """
-        optional n × m array, representing the internal state of the m level system of each atom
+        optional n × 1 x m array, representing the ket vectors of the m level system of each atom
         """
-        return self._state_vectors
+        return self._state_kets
 
-    @state_vectors.setter
-    def state_vectors(self, new_state_vectors):
-        new_state_vectors = np.array(new_state_vectors)
-        if new_state_vectors.ndim == 1:
+    @state_kets.setter
+    def state_kets(self, new_state_kets):
+        if isinstance(new_state_kets, list):
+            new_state_kets = np.array([new_state_kets]).T

```

Comment:
same thing as in __init__: the transpose does nothing

Reply:
thats not true.
if you want to set the state_ket with the list new_state_vectors = [1,0], then the np.array([new_state_kets]) part maks it a numpy array of shape (2,1) and the transpose makes it an actual row vector

Reply:
sorry, it is first (1,2) and transpose makes it (2,1)

Reply:
true, didn't see the additional list in `[new_state_kets]`